deploy/helm/sumologic/templates/logs/collector/otelcol/service.yaml

```diff
@@ -0,0 +1,27 @@
+{{- if .Values.otellogs.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "sumologic.metadata.name.logs.collector.service" . }}
+  labels:
+    app: {{ template "sumologic.labels.app.logs.collector.service" . }}
+    {{- include "sumologic.labels.scrape.logs.collector" . | nindent 4 }}
+    {{- include "sumologic.labels.common" . | nindent 4 }}
+{{- if .Values.otelcol.serviceLabels }}
+{{ toYaml .Values.otelcol.serviceLabels | indent 4 }}
+{{- end }}
+spec:
+  selector:
+    app.kubernetes.io/name: {{ template "sumologic.labels.app.logs.collector.pod" . }}
+  ports:
+  - name: pprof
+    port: 1777
+    targetPort: 1777
+    protocol: TCP
```

Comment:
I don't think this is particularly useful. You're always profiling some specific Pod.

Reply:
I can drop it